test/lib/CompilerManager.test.js

```diff
@@ -3,6 +3,16 @@
 const CompilerManager = require("../../lib/CompilerManager");
 const { FIRST_BUILD, BUILDING, ERROR, DONE } = require("../../lib/constants");
 
+let consoleSpy;
+
+beforeAll(() => {
+    consoleSpy = jest.spyOn(global.console, "error").mockImplementation(() => {});
+});
+
+afterAll(() => {
+    consoleSpy.mockRestore();
+});
+
```

Comment:
fwiw, I don't think you'd need this `consoleSpy` variable
```suggestion
beforeAll(() => {
    jest.spyOn(global.console, "error").mockImplementation(() => {});
});

afterAll(() => {
    global.console.error.mockRestore();
});

```